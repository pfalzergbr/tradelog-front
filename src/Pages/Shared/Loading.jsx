import React from 'react';
import LoadingSpinner from '../../assets/animations/loading-spinner.gif';

const Loading = props => {
  return (
    <div>
      <img className="loading-spinner" src={LoadingSpinner}alt="Animated loading spinner"/>
    </div>
  );
};

export default Loading;
