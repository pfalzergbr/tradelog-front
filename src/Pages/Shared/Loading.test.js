import { shallow } from 'enzyme';
import Loading from './Loading';
import React from 'react';

it('should render Loading Component', () => {
    expect(shallow(<Loading />)).toMatchSnapshot();
})