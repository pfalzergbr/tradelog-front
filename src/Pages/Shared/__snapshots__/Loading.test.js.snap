// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Loading Component 1`] = `
<div>
  <h1>
    Loading...
  </h1>
</div>
`;
